lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1545,3 +1545,57 @@ CompilerInvocation::loadFromSerializedAST(StringRef data) {
                         extendedInfo.getExtraClangImporterOptions().end());
   return info.status;
 }
+
+bool CompilerInvocation::getOrCreatePCH() {
+
+  const auto &BridgingHeader = getClangImporterOptions().BridgingHeader;
+
+  const auto &PCHOutputDir = getClangImporterOptions().PrecompiledHeaderOutputDir;
+
+  if (BridgingHeader.empty() || PCHOutputDir.empty()) {
+    return true;
+  }
+
+  CompilerInstance ReadInstance;
+  auto FailedToGetReadInstance = ReadInstance.setup(*this);
```

Comment:
Once  ClangImporter::create is called, isn't it too late to emit a PCH and pull it in? Or is the bridging header imported later?